website/docs/docs/cloud/manage-access/about-access.md

```diff
@@ -31,14 +31,12 @@ user can only have one type of license at any given time.
 
 A user's license type controls the features in dbt Cloud that the user is able
 to access. dbt Cloud's three license types are:
- - **Read Only**
- - **Developer**
- - **IT**
+
+ - **Developer** &mdash; User may be granted _any_ permissions.
```

Comment:
consolidating paragraph into bullets so its easier to scan and less reading for user